examples/core/instancing/app.js

```diff
@@ -154,13 +162,36 @@ class AppAnimationLoop extends AnimationLoop {
       }
     });
 
-    return {statsWidget};
+    const memWidget = new StatsWidget(statsManager.get('Memory Usage'), {
```

Comment:
Side note: My initial thinking is that we would have one widget that can show multiple screens and can be expanded/collapsed/circulated by the user, rather than add multiple widgets. Zero config in apps and all power in the users hands to slice and dice the data. Such sophistication would come later of course.